ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -102,6 +102,70 @@ def _get_table_description(schema: str, table: str, inspector: Inspector) -> str
     return description
 
 
+def get_dbt_details(config):
+    dbt_catalog = "{}"
+    dbt_manifest = "{}"
+    if config.dbtProvider:
+        if config.dbtProvider.value == "s3":
+            from metadata.utils.aws_client import AWSClient
+
+            aws_client = AWSClient(config.dbtConfig).get_resource("s3")
+            buckets = aws_client.buckets.all()
+            for bucket in buckets:
+                for bucket_object in bucket.objects.all():
+                    if "manifest_1.0" in bucket_object.key:
```

Comment:
We should use some constants for "manifest_1.0" and catalog, as we are repeating them a lot

Reply:
yes I'll use a constant